### PR TITLE
Point to Options in local setup doc.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**

### CODE REVIEW

File: docs/SETUP_LOCAL.md [hunk 1]
```diff
@@ -54,11 +54,7 @@
 
   We recommend you to subscribe to changes in this repo (follow it on GitHub / watch for tweets of its maintainers / etc). This will allow you to apply best practises we found to make `mainframer` faster and safer.
 
-2. Add the following content to `.mainframer/config`.
-
-  ```properties
-  remote_machine={REMOTE_MACHINE_NAME}
-  ```
+2. Carefully read [Options documentation](OPTIONS.md) and configure mainframer for *your project*.
```
Comment (arturdryomov) [inline docs/SETUP_LOCAL.md]: `your kind of project` without italic?

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: docs/SETUP_LOCAL.md (hunk 1 above)
Comment (artem-zinnatullin) [reply docs/SETUP_LOCAL.md]: I would keep italic and use your phrase.

File: docs/SETUP_LOCAL.md [hunk 2]
```diff
@@ -54,11 +54,7 @@
 
   We recommend you to subscribe to changes in this repo (follow it on GitHub / watch for tweets of its maintainers / etc). This will allow you to apply best practises we found to make `mainframer` faster and safer.
 
-2. Add the following content to `.mainframer/config`.
-
-  ```properties
-  remote_machine={REMOTE_MACHINE_NAME}
-  ```
+2. Carefully read [Options documentation](OPTIONS.md) and configure mainframer for *your kind of project*.
```
Comment (arturdryomov) [inline docs/SETUP_LOCAL.md]: Without italic.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: docs/SETUP_LOCAL.md (hunk 2 above)
Comment (artem-zinnatullin) [reply docs/SETUP_LOCAL.md]: ARh, k

**Review by arturdryomov — APPROVED** [review]

**Pull request merged**